source/administration/devops-command-center.rst

```diff
@@ -64,48 +64,126 @@ Viewing and creating playbooks
 
 Select **!** in the channel header to open the RHS panel. Select the **Playbooks** icon to open the playbooks backstage and list all playbooks associated with the current team. Select **+ New Playbook** to start a new playbook and checklist. Enter a playbook name and choose **Save Playbook**.
 
-Creating checklists
-~~~~~~~~~~~~~~~~~~~
+Public and private incidents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Checklists are created within a playbook and provide steps to follow during an incident. Checklists can be created ahead of time as part of a playbook, or during an incident.
+Incidents can be:
 
-Editing checklists
-~~~~~~~~~~~~~~~~~~~
+- **Public:** Anyone in the team can view the incident details and join the incident channel.
+- **Private:** Only incident members who were in the incident when it was started have access to the ongoing incident and view incident details after the incident has ended.
+
+**Creating a public incident channel**
 
-You can add steps and edit checklist items during an active incident. You can also edit checklist items in playbooks that are not being used in active incidents.
+1. Navigate to **Main Menu > Incidents & Playbooks > Playbooks**.
+2. Select **+ New Playbook**.
+3. Enter a name for the playbook.
+4. Move the toggle to the right to **Create Public Incident**.
+5. Add a few checklist items.
+6. Select **Save**.
+7. Switch to the main channel view.
+8. Create an incident by selecting the playbook you just created.
 
-To edit a checklist, choose **Edit**.
+**Creating a private incident channel**
 
-- Select the text field to edit the item details.
-- Hover over an input box and select the **X** to remove the checklist item.
-- Drag the hamburger menu to rearrange the checklist items.
+1. Navigate to **Main Menu > Incidents & Playbooks > Playbooks**.
+2. Select **+ New Playbook**.
+3. Enter a name for the playbook.
+4. Keep the **Create Public Incident** option disabled.
+5. Add a few checklist items.
+6. Select **Save**.
+7. Switch to the main channel view.
+8. Create an incident by selecting the playbook you just created.
```

Comment:
Just an idea: maybe we can focus this on playbooks instead of incidents, because `Creating a public/private incident channel` only depends on the playbook selected.

Reply:
That's a good idea, can definitely do that.

Reply:
@agarciamontoro, I've done a minor change to this to bring it in line with your feedback. Once this is merged I'll do a second review and rewrite.